schema.sql

```diff
@@ -1,5 +1,17 @@
-/* Database schema to keep the structure of entire database. */
+-- This part is for creating the animals table
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE IF NOT EXISTS public.animals
+(
+    id integer NOT NULL GENERATED ALWAYS AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 ),
+    name "char",
```

Comment:
- [ ] Good job @Rafiduzzaman please use the `varchar` instead of `char` because varchar has more flexibility and doesn't contain the unnecessary space as `char` does, and `char` type is not flexible too. For more information you can learn on official website.